venter/views.py

```diff
@@ -18,6 +19,36 @@
 from venter.serializers import CommentPostSerializer
 from venter.serializers import CommentSerializer
 
+class RelevantViewSet(viewsets.ModelViewSet):
+    queryset = Complaints.objects.all()
+    serializer_class = ComplaintSerializer
+
+    @classmethod
+    def list(cls, request):
+        """RelevantViewSet for the getting list of relevant complaints"""
+        queryset = Complaints.objects.all()
+        serialized = queryset.values_list('description')
```

Comment:
Why is this returning a list of descriptions? Also this call is killing performance here, since this is a database call to get the whole table.